content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -135,7 +135,7 @@ See section [specifying your own pod selector](#specifying-your-own-pod-selector
 
 ### Parallel execution for Jobs {#parallel-jobs}
 
-There are three main types of task suitable to run as a Job:
+There are three main types of tasks suitable to run as a Job:
```

Comment:
The original grammar is more idiomatic English @phumberdroz - it is the compound _types of task_ that is plural here.

Reply:
👍

Reply:
If you could omit this change, the others I think are good to go through.

Reply:
Hi @phumberdroz I think if changes on this line is omitted this pr is good to go